misc-scripts/xref_mapping/XrefParser/ChecksumParser.pm

```diff
@@ -71,7 +71,7 @@ sub _transfer_contents {
   while(my $line = <$input_fh>) {
     chomp $line;
     my ($upi, $checksum) = split(/\s+/, $line);
-    my @output = ($counter++, $source_id, $upi, $checksum);
+    my @output = (++$counter, $source_id, $upi, $checksum);
```

Comment:
I'm not a fan of using ++ except when it's the only thing happening on a line or in cases like for loops where it's idiomatic. It's hard to read and can lead to subtle bugs like the one that's fixed here. Would prefer it added standalone above. However, as this is three commits down in the history will leave it up to you to change or not.

Reply:
At first I wanted to avoid changing the code more than necessary - but heck, I don't like in-line increments either and we might as well do it properly. Done.